Client/.gitignore

```diff
@@ -0,0 +1,2 @@
+dist
```

Comment:
Client folder needed a .gitignore. Ignoring the dist and auto generated Relay files. Those would get created as part of Continuous Deployment.